packages/vertexai/src/methods/generate-content.ts

```diff
@@ -59,7 +59,7 @@ export async function generateContent(
     requestOptions
   );
   const responseJson: GenerateContentResponse = await response.json();
-  const enhancedResponse = addHelpers(responseJson);
+  const enhancedResponse = createEnhancedContentResponse(responseJson);
```

Comment:
Did you assign it to a named variable first here and in the other places to increase readability? That's fine with me, seems reasonable, just wondering.

Reply:
I think it improves readability only slightly, but the main reason I like this style because it makes it easier to debug since I can step over and see the return value before returning to the calling function, and I can log the value before it's returned without having to move stuff around.